Lib/Src/CUartCom.cpp

```diff
@@ -10,6 +10,8 @@
 
 #include "CUartCom.h"
 
+#define BYTE 1
```

Comment:
I realise you are trying to avoid "magic numbers" problem, but in this case, it does not add to clarity, but obfuscates that you are sending one byte at a time.